### ISSUE TITLE
NoxPlayer A12 finally have zygisk

### ISSUE THREAD

**Issue by VisionR1** [HuskyDG/magisk-files#199]
@HuskyDG this is the solution for latest Nox Player Android 12 with latest magisk delta and this module https://github.com/Dr-TSNG/ZygiskNext/releases/tag/v4-0.8.1 without enable the inside magisk zygisk and works and the LSPosed

![Magisk-0fe46c5a-delta(26301)](https://github.com/HuskyDG/magisk-files/assets/25982450/6c3f7189-57f3-4f7c-a5b2-8babf33e431d)
![Zygisk Next](https://github.com/HuskyDG/magisk-files/assets/25982450/0d55eb77-edbd-4f4b-b94e-379372ceddee)
![Zygisk-LSPosed](https://github.com/HuskyDG/magisk-files/assets/25982450/990e535e-b139-4883-ab66-4b4ea8ddfce7)

**Comment by Jordan231111** [comment]
Yep its working, this is really nice now its working, ty for ur efforts in testing what works

**Comment by directentis1** [comment]
Thanks, i'm happy that I can continue use LSPosed while hide zygisk from banking apps.. Hope it lasts a long time 😂

**Comment by VisionR1** [comment]
Yeah, all thanks must go to @Nullptr and @5ec1cff for this great module with working with latest nox player android 12, keep the good work

**Comment by VisionR1** [comment]
Yeah, all thanks must go to @Dr-TSNG and @5ec1cff for this great module with working with latest nox player android 12, keep the good work

**Comment by directentis1** [comment]
Fortunately, my banking apps can't detect, but, momo root checking app detect magisk after enable zygisk-next module.
Perhaps there is still more work to be done.
But really thank those people for all their efforts so far!

![](https://i.imgur.com/x2hrtgZ.png)

**Comment by VisionR1** [comment]
> Fortunately, my banking apps can't detect, but, momo root checking app detect magisk after enable zygisk-next module. Perhaps there is still more work to be done. But really thank those people for all their efforts so far!
> 
> ![](https://camo.githubusercontent.com/e1b023853b89900d897ef3ed2b28bf0b99ca0a064249079e039d40ec90ad4c92/68747470733a2f2f692e696d6775722e636f6d2f7832687274675a2e706e67)

Oh, understand only in emulator have test it, in resl phone maybe this work

![IMG_20231028_222350](https://github.com/HuskyDG/magisk-files/assets/25982450/b6d53c06-ad7d-4a85-9884-7c7d0e88bbf4)

**Comment by VisionR1** [comment]
Post in thread '🔧 [MODULE] Play Integrity Fix (SafetyNet fix)' https://xdaforums.com/t/module-play-integrity-fix-safetynet-fix.4607985/post-89123880

**Comment by SuperDuper77** [comment]
What ZygiskNext for ?

**Comment by VisionR1** [comment]
> What ZygiskNext for ?

Standalone implementation of Zygisk, providing Zygisk API support for KernelSU and a replacement of Magisk's built-in Zygisk

More information here: https://github.com/Dr-TSNG/ZygiskNext

**Comment by pogchamppudge** [comment]
![image](https://github.com/HuskyDG/magisk-files/assets/149511443/66c57a6e-f575-4e8a-b3df-72fb126656c5)
![image](https://github.com/HuskyDG/magisk-files/assets/149511443/6c674776-7d42-4d88-be0e-aeed0de6f4df)
Is it possible to remove these errors? Nox a12. I used to be able to remove them all without problems, but the magisk specific check always gets an error. Now I have installed the latest version of delta and zigysk next, now I get other errors.

**Comment by VisionR1** [comment]
> ![image](https://user-images.githubusercontent.com/149511443/279432741-66c57a6e-f575-4e8a-b3df-72fb126656c5.png) ![image](https://user-images.githubusercontent.com/149511443/279432877-6c674776-7d42-4d88-be0e-aeed0de6f4df.png) Is it possible to remove these errors? Nox a12. I used to be able to remove them all without problems, but the magisk specific check always gets an error. Now I have installed the latest version of delta and zigysk next, now I get other errors.

I think because is run on emulator, and detected the SU binary from the integrated nox player

**Comment by VisionR1** [comment]
For now, i have no clue, if found something, i write here

**Comment by ccooddeeaa** [comment]
Nox12

![image](https://github.com/HuskyDG/magisk-files/assets/148022005/d03edf4e-3175-4d28-b5cb-4bbe5ff08664)
![image](https://github.com/HuskyDG/magisk-files/assets/148022005/97ec2988-0f40-423e-a23f-379a701f7dbd)
![image](https://github.com/HuskyDG/magisk-files/assets/148022005/e66e8241-8ce7-4d0e-95a2-969571cce0f2)
![image](https://github.com/HuskyDG/magisk-files/assets/148022005/495a17e0-dfc3-4997-9aba-276951307df0)


Thanks for this module @Dr-TING and @5ec1cff

the result using MagiskFrida (for those who understand what it is)

**Comment by Jordan231111** [comment]
Do u mean this module https://github.com/ViRb3/magisk-frida

**Comment by ccooddee-masters** [comment]
Yes, this one.  already updated, I have the previous 16.1.4-1

**Comment by Jordan231111** [comment]
> For now, i have no clue, if found something, i write here

mumu android 12 was released a couple days ago
have u tested that, is there any way to get magisk/zygisk running there as it is overall a much better emulator

**Comment by VisionR1** [comment]
> > For now, i have no clue, if found something, i write here
> 
> mumu android 12 was released a couple days ago have u tested that, is there any way to get magisk/zygisk running there as it is overall a much better emulator

First time hear it, is better than BlueStacks Android 11 and nox player android 12 ?

**Comment by Jordan231111** [comment]
It has been the best performance emulator for many years now tbh @VisionR1

**Comment by Jordan231111** [comment]
Nox 12 background loading conflicts with magisk and will cause lag which i have noticed a lot of overall kinda buggy

**Comment by Jordan231111** [comment]
but with mumu 12 u can manually select if u want it or not,
unfortunately it seems like even with write disk the magisk with zygisk cant install correctly even with zygisk next
wonder if i can still get riru on it or some other stuff if u have time to do more testing

**Comment by VisionR1** [comment]
> Nox 12 background loading conflicts with magisk and will cause lag which i have noticed a lot of overall kinda buggy

Yeah, now you mentioned this, i think have the same bug

> but with mumu 12 u can manually select if u want it or not, unfortunately it seems like even with write disk the magisk with zygisk cant install correctly even with zygisk next wonder if i can still get riru on it or some other stuff if u have time to do more testing
> 
> Make sure u dont turn on background loading in mumu12 while testing

Oh, understand, tomorrow i download and install the mumu and testing and write the results here

**Comment by Jordan231111** [comment]
> > Nox 12 background loading conflicts with magisk and will cause lag which i have noticed a lot of overall kinda buggy
> 
> Yeah, now you mentioned this, i think have the same bug
> 
> > but with mumu 12 u can manually select if u want it or not, unfortunately it seems like even with write disk the magisk with zygisk cant install correctly even with zygisk next wonder if i can still get riru on it or some other stuff if u have time to do more testing
> > Make sure u dont turn on background loading in mumu12 while testing
> 
> Oh, understand, tomorrow i download and install the mumu and testing and write the results here

tysm since u have more understanding of some stuff and my basic tests seem to fail, maybe need to play around a lot
such as changing the device model setting too

**Comment by VisionR1** [comment]
> > > Nox 12 background loading conflicts with magisk and will cause lag which i have noticed a lot of overall kinda buggy
> > 
> > 
> > Yeah, now you mentioned this, i think have the same bug
> > > but with mumu 12 u can manually select if u want it or not, unfortunately it seems like even with write disk the magisk with zygisk cant install correctly even with zygisk next wonder if i can still get riru on it or some other stuff if u have time to do more testing
> > > Make sure u dont turn on background loading in mumu12 while testing
> > 
> > 
> > Oh, understand, tomorrow i download and install the mumu and testing and write the results here
> 
> tysm since u have more understanding of some stuff and my basic tests seem to fail, maybe need to play around a lot such as changing the device model setting too

Understand, tomorrow as i say, and i try to testing some modifications and see if work and write my results here

**Comment by Jordan231111** [comment]
in case u can't find it as people always get confused between this and memu lol or where to download

https://www.mumuplayer.com/update/

**Comment by VisionR1** [comment]
> in case u can't find it as people always get confused between this and memu lol or where to download
> 
> https://www.mumuplayer.com/update/

Thanks, go download and install

**Comment by VisionR1** [comment]
@Jordan231111 First try with Riru v25.4.4 and LSPosed-v1.8.6 riru. MuMu setting root always enabled and write disk

![MuMu Riru](https://github.com/HuskyDG/magisk-files/assets/25982450/4efd0e6e-daa7-4915-a9cd-6efe677abfd1)

**Comment by k4karoto** [comment]
![rootbeee](https://github.com/HuskyDG/magisk-files/assets/150601270/f041fe54-c753-41ea-8075-67de3c0cd4a4)

I did some tests, but with no success, for anyone trying, I used magisk delta 26100, it installs normally
however it does not activate zygisk, it stops starting the emulator
Riru installs it, but it stays like this, does anyone know a way to bypass it this way?

**Comment by k4karoto** [comment]
Does anyone know a way to tell applications that the device is not an emulated device?
It even works on another emulator

**Comment by ccooddee-masters** [comment]
> Does anyone know a way to tell applications that the device is not an emulated device? It even works on another emulator

root can be hidden on any emulator using frida.
I myself am looking for a method to hide from applications that determine that this is an emulator.
I know that this can also be done using frida, but I need to disassemble each application separately. now I'm trying to figure it out

**Comment by ccooddee-masters** [comment]
> ![rootbeee](https://user-images.githubusercontent.com/150601270/282304196-f041fe54-c753-41ea-8075-67de3c0cd4a4.png)
> 
> I did some tests, but with no success, for anyone trying, I used magisk delta 26100, it installs normally however it does not activate zygisk, it stops starting the emulator Riru installs it, but it stays like this, does anyone know a way to bypass it this way?

to activate Zygisk in 26100, use the Zygisk Next module. After that install Lsposed and you will see that it works

**Comment by VisionR1** [comment]
> > I did some tests, but with no success, for anyone trying, I used magisk delta 26100, it installs normally however it does not activate zygisk, it stops starting the emulator Riru installs it, but it stays like this, does anyone know a way to bypass it this way?
> 
> to activate Zygisk in 26100, use the Zygisk Next module. After that install Lsposed and you will see that it works

You mean the Magisk Delta Stable v26.1 and what version of this module? Because the latest need Magisk version 26300

**Comment by Jordan231111** [comment]
> > I did some tests, but with no success, for anyone trying, I used magisk delta 26100, it installs normally however it does not activate zygisk, it stops starting the emulator Riru installs it, but it stays like this, does anyone know a way to bypass it this way?
> 
> to activate Zygisk in 26100, use the Zygisk Next module. After that install Lsposed and you will see that it works

i dont believe this works for mumu 12 as i tested a lot with zygisk next unless there is a specifc version that can be exploited

**Comment by VisionR1** [comment]
> > > I did some tests, but with no success, for anyone trying, I used magisk delta 26100, it installs normally however it does not activate zygisk, it stops starting the emulator Riru installs it, but it stays like this, does anyone know a way to bypass it this way?
> > 
> > 
> > to activate Zygisk in 26100, use the Zygisk Next module. After that install Lsposed and you will see that it works
> 
> i dont believe this works for mumu 12 as i tested a lot with zygisk next unless there is a specifc version that can be exploited

Yeah, not working, i try lot all stuck in boot screen

**Comment by Kaan3801** [comment]
> @Jordan231111 First try with Riru v25.4.4 and LSPosed-v1.8.6 riru. MuMu setting root always enabled and write disk
> 
> ![MuMu Riru](https://user-images.githubusercontent.com/25982450/282301552-4efd0e6e-daa7-4915-a9cd-6efe677abfd1.png)

I've been trying for hours. Thanks.

**Comment by k4karoto** [comment]
My dears, getting away from this subject a little in the emulator, does anyone know a way to make the cell phone camera pass images from a camera emulator, like ManyCam

**Comment by VisionR1** [comment]
> My dears, getting away from this subject a little in the emulator, does anyone know a way to make the cell phone camera pass images from a camera emulator, like ManyCam

Download ManyCam to the emulator and in your smartphone from Google play store and connect it between if have a options, something like that

**Comment by k4karoto** [comment]
> smartphone

I wanted to change the camera to Manycam, the same way it works in the emulator

**Comment by VisionR1** [comment]
> > smartphone
> 
> I wanted to change the camera to Manycam, the same way it works in the emulator

If you mean something like default camera change it with ManyCam in your phone, Google it for info

**Comment by k4karoto** [comment]
> > > smartphone
> > 
> > 
> > I wanted to change the camera to Manycam, the same way it works in the emulator
> 
> If you mean something like default camera change it with ManyCam in your phone, Google it for info

How could I do this? for example, if I want to make a video call, I would like to put an old video of me and the other person on the other side, see my old video, this using my cell phone, it has the latest magisk and zygisk enabled

**Comment by VisionR1** [comment]
> > > > smartphone
> > > 
> > > 
> > > I wanted to change the camera to Manycam, the same way it works in the emulator
> > 
> > 
> > If you mean something like default camera change it with ManyCam in your phone, Google it for info
> 
> How could I do this? for example, if I want to make a video call, I would like to put an old video of me and the other person on the other side, see my old video, this using my cell phone, it has the latest magisk and zygisk enabled

Understand, unfortunately I'm not special with this you ask, Google it to found more info about it or ask in xda forum
